build/root/Makefile

```diff
@@ -46,12 +46,12 @@ GENERATED_FILE_PREFIX := zz_generated.
 # Metadata for driving the build lives here.
 META_DIR := .make
 
-# Our build flags.
-# TODO(thockin): it would be nice to just use the native flags.  Can we EOL
-#                these "wrapper" flags?
-KUBE_GOFLAGS := $(GOFLAGS)
-KUBE_GOLDFLAGS := $(GOLDFLAGS)
-KUBE_GOGCFLAGS = $(GOGCFLAGS)
+ifdef KUBE_GOFLAGS
+$(info KUBE_GOFLAGS is deprecated. Please use GOFLAGS instead)
+ifndef GOFLAGS
+GOFLAGS := $(KUBE_GOFLAGS)
```

Comment:
@thockin Now i remember why i gave up on that. The instructions in this make file gets executed twice, in the first pass the GO_FLAGS get set to what's in KUBE_GOFLAGS and the second pass errors out. Any way to avoid that?

See experiment in http://paste.openstack.org/show/611567/